api/schemas/input/file.json

```diff
@@ -1,18 +1,19 @@
 {
   "$schema": "http://json-schema.org/draft-04/schema#",
   "type": "object",
+  "oneOf": [
+      { "$ref": "file.json#/definitions/MR" },
+      { "$ref": "file.json#/definitions/physio" },
+      { "$ref": "file.json#/definitions/CT" }
+  ],
```

Comment:
It allows the instrument and measurements fields to be merged with the rest of the fields in properties. Otherwise you could add the oneOf restraint to a field in properties, in which case instrument and measurements would have to be a named json object